Buildings/Controls/OBC/ASHRAE/G36_PR1/package.mo

```diff
@@ -2,16 +2,45 @@ within Buildings.Controls.OBC.ASHRAE;
 package G36_PR1 "Package with control sequences from ASHRAE Guideline 36"
   extends Modelica.Icons.VariantsPackage;
 
+
+
+
 annotation (Documentation(info="<html>
 <p>
 This package contains control sequences from
-ASHRAE Guideline 36.
+ASHRAE Guideline 36 (G36).
 All sequences are created using blocks from the
 <a href=\"modelica://Buildings.Controls.OBC.CDL\">
 Buildings.Controls.OBC.CDL</a> library, following the
 <a href=\"http://obc.lbl.gov/specification/cdl.html\">
 CDL specification</a>.
 </p>
+<p>
+The G36 library is structured as follows:
+<ul>
+<li><a href=\"modelica://Buildings.Controls.OBC.ASHRAE.G36_PR1.AHUs\">
+Buildings.Controls.OBC.ASHRAE.G36_PR1.AHUs</a> contains control sequences
+for generating controller setpoints, e.g. supply air temperature <code>TSup</code>,  
+and actuation signals for mechanical elements of an AHU, e.g. outdoor air damper 
+position <code>yOutDamPos</code>.
+</li>
+<li><a href=\"modelica://Buildings.Controls.OBC.ASHRAE.G36_PR1.Constants\">
+Buildings.Controls.OBC.ASHRAE.G36_PR1.Constants</a> is a library of status
+signals used in the sequences for communicating system operation modes, freeze 
+protections status, zone state, etc.
+</li>
+<li><a href=\"modelica://Buildings.Controls.OBC.ASHRAE.G36_PR1.Generic\">
+Buildings.Controls.OBC.ASHRAE.G36_PR1.Generic</a> stores any sequences with
+outputs utilized accross the HVAC system, both for the AHU and the 
+terminal unit control.
+</li>
+<li><a href=\"modelica://Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits\">
+Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits</a> contains control
```

Comment:
This should say that this package is only for terminal unit controller and set points